dtcli/scripts/dt.py

```diff
@@ -389,6 +403,62 @@ def upload(**kwargs):
 
 
 
+@extension.command(
+    help="Downloads all schemas from choosen version"
+)
+@click.option(
+    "--version", prompt=True, help="Schema Version e.g. 1.235"
+)
+@click.option(
+    "--tenant-url", prompt=True, help="Dynatrace environment URL, e.g., https://<tenantid>.live.dynatrace.com"
+)
+@click.option(
+    "--api-token", prompt=True, help="Dynatrace API token. Please note that token needs to have the 'Write extension' scope enabled."
+                                     "If using token from file, type 'file'."
+)
+@click.option(
+    "--token-path",
+    default=DEFAULT_TOKEN_PATH, show_default=True,
+    help="Directory where token file can be found. Please note that token needs to have the 'Write extension' scope enabled.",
+)
+@click.option(
+    "--download-dir",
+    default=DEFAULT_DOWNLOAD_DIR, show_default=True,
+    help="Directory where folder schemas will be created with all downloaded files",
+)
+def schemas(**kwargs):
+    if kwargs['api_token'] != "file":
+        token = kwargs['api_token']
+    elif token_load(kwargs['token_path']):
+        token = token_load(kwargs['token_path'])
+    else:
+        token = os.getenv('TOKEN')

```

Comment:
Can we do something more specific than just "TOKEN"? Like: "DT_TENANT_API_TOKEN" (which can be used verbatim, but it's just an example demonstrating how a more specific name can be made)

Reply:
Changed to "DTCLI_API_TOKEN"